### PR TITLE
location-verification Create Location Verification User Story

### LINKED ISSUE #220: Provide API readiness checklist
**Problem description**
To release an API version, an API Sub Project needs to provide the required release assets as specified in the API Readiness Checklist. Source [here](https://wiki.camaraproject.org/display/CAM/API+Release+Process#APIReleaseProcess-APIreadinesschecklist) §API readiness checklist

The template for the checklist is the [API-Readiness-Checklist.md](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md) file located in the CAMARA / ReleaseManagement / documentation repository.

**Expected action**
Create md file checklist

**Additional context**
@jlurien @maxl2287  I've created one issue but we'll need 3 files - one for each API - If we prefer to have 3 separate issues I'm fine with it.

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

Including user story for Location Verification API

#### Which issue(s) this PR fixes:

Fixes #220 

#### Changelog input

```
Include user story for Location Verification API

```

### CODE REVIEW

File: documentation/SupportingDocuments/Location_Verification_User_Story.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to verify the location of a user's device, so that I can ensure the device is within a specified area for security and contextual-based services. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br> - Verifies if the specified device is located within a predefined geographical area (circle with given coordinates and radius). |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:BusinessManager has successfully subscribed to the Location Verification API product from the product catalog.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer:User performs an authorization request to CSP.</li><li>The Customer:User has the access token allowing a secure access of the API.|
```
Comment (bigludo7) [inline documentation/SupportingDocuments/Location_Verification_User_Story.md:8]: CSP: probably we need to provide full name for first use in the user story. BTW CSP should not be in the `Actors `part?

File: documentation/SupportingDocuments/Location_Verification_User_Story.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+| ***Activities/Steps*** | **Starts when:** The customer application makes a POST request to the Location Verification API providing the device identifier (e.g., phone number) and the area information (center coordinates and radius).<br>**Ends when:** The API server responds with the verification result indicating if the device is within the specified area, partially matches the area, or is unknown. |
```
Comment (bigludo7) [inline documentation/SupportingDocuments/Location_Verification_User_Story.md:9]: The customer also asks for a freshness of the information - worth to add it perhaps

**Review by bigludo7 — COMMENTED** [review]
Hello @jgarciahospital 
2 small comments for your consideration.

File: documentation/SupportingDocuments/Location_Verification_User_Story.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Location Verification API User Story
+
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to verify the location of a user's device, so that I can ensure the device is within a specified area for security and contextual-based services. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br> - Verifies if the specified device is located within a predefined geographical area (circle with given coordinates and radius). |
```
Comment (alpaycetin74) [inline documentation/SupportingDocuments/Location_Verification_User_Story.md:7]: Do we need to specify the area is a circle? We can support other shapes in the future. Then we would need to come back and update the story for every new shape. But if you find it OK, that's OK for me too. Thank you.

**Comment by jlurien** [comment]
@bigludo7 @alpaycetin74 please review if your comments and questions have been addressed. Thanks

**Review by alpaycetin74 — APPROVED** [review]
LGTM, thank you

**Review by bigludo7 — APPROVED** [review]
LGTM

**Pull request merged**